tools/junitreport/pkg/parser/gotest/data_parser_test.go

```diff
@@ -161,33 +161,33 @@ func TestExtractSuiteName(t *testing.T) {
 		fail         bool
 	}{
 		{
-			name:         "basic",
-			testLine:     "ok  	package/name 0.160s",
+			name: "basic",
```

Comment:
Looks like this is formatted with go1.18. @pacevedom the codebase has been recently formatted with go1.19.